ftw/django_utils/settings/logging.py

```diff
@@ -0,0 +1,100 @@
+import sys
+
+import sentry_sdk
+from configurations import values
+from sentry_sdk.integrations.django import DjangoIntegration
+
+
+class LogMixin:
+    """
+    This mixins contains the logging configuration of our Django based
+    web applications. Its default values are suited for production.
+    """
+
+    @property
+    def LOGGING_FILENAME(self):
+        default_value = super().BASE_DIR / "log" / "django.log"
+        return values.Value(default=default_value, environ_name="LOGGING_FILENAME")
+
+    LOGGING_LEVEL = values.Value("WARNING")
```

Comment:
I was wondering if we ever configured the log level through an environment variable or if we could ditch this?

Reply:
It's nice to have this if you have a weird bug to enable `DEBUGGING` level logging.